draft-ietf-cbor-7049bis.md

```diff
@@ -715,16 +720,19 @@ information is in the initial bytes.
 |          25 | IEEE 754 Half-Precision Float (16 bits follow)            |
 |          26 | IEEE 754 Single-Precision Float (32 bits follow)          |
 |          27 | IEEE 754 Double-Precision Float (64 bits follow)          |
-|       28-30 | (Unassigned)                                              |
+|       28-30 | Unassigned, currently not well-formed                     |
```

Comment:
I know what you are trying to do, but the "currently" sounds weird in what is trying to be a STD.  Maybe "not well-formed now".

Reply:
I'm just trying to make @paulehoffman happy from https://github.com/cbor-wg/CBORbis/pull/17#issuecomment-454463966. 🤣 I've taken your phrasing, but "in this version of CBOR" might also work.